token-service/src/main/java/org/p2p/token/service/repository/TokenServiceLocalRepository.kt

```diff
@@ -0,0 +1,13 @@
+package org.p2p.token.service.repository
+
+import org.p2p.token.service.model.TokenServiceMetadata
+import org.p2p.token.service.model.TokenServiceNetwork
+import org.p2p.token.service.model.TokenServicePrice
+
+interface TokenServiceLocalRepository {
+    fun setTokensPrice(networkChain: TokenServiceNetwork, prices: List<TokenServicePrice>)
+    fun findTokenPriceByAddress(networkChain: TokenServiceNetwork, address: String): TokenServicePrice?
```

Comment:
be careful, it already violates SRP, maybe it would be a good idea to split into two repositories
because one day we may move to a database / prefs for prices or metadata